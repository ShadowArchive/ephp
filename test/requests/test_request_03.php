<?php

foreach ($_REQUEST as $k => $v) {
    print "REQUEST[$k] => $v\n";
}
