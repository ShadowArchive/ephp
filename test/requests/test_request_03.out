REQUEST[A] => 12
REQUEST[B] => 24
